docs/guide/api-javascript.md

```diff
@@ -15,7 +15,7 @@ async function createServer(inlineConfig?: InlineConfig): Promise<ViteDevServer>
 ```js
 const { createServer } = require('vite')
 
-;(async () => {
```

Comment:
These semiconlons are necessary because the previous line is not ending with it.